### ISSUE TITLE
Project view; some elements do not give clear indications of being highlighted when tabbed to

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2334]
**Describe the bug**

In the project view, the several buttons do not give proper indication of being highlighted:

``Go back`` button doesn't have any obvious highlight / indication

top menu bar (toggle between list and card view, as well as open / duplicate / delete when a project is selected) have slight highlights, but they are very subtle / not easy to see

**To Reproduce**
1. Go to project view
2. tab through different buttons, and see that some are hard to find

**Expected behavior**

should have highlight / indication of selected item

**Screenshots**

![2019-08-02 14 50 18](https://user-images.githubusercontent.com/5615930/62400850-90dff000-b535-11e9-9166-92c3b4f42ad9.gif)

**Comment by jwunderl** [comment]
the go back button not highlighting applies everywhere it is found; more commonly seen in the extensions dialogue
